core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/support/IncludeExclude.java

```diff
@@ -250,13 +307,27 @@ public IncludeExclude(long[] includeValues, long[] excludeValues) {
         this(convertToBytesRefSet(includeValues), convertToBytesRefSet(excludeValues));
     }
 
+    public IncludeExclude(int partition, int pOf) {
+        this.incZeroBasedPartition = partition - 1;
```

Comment:
Check that partition >= 1 && pof > partition ? It's only checked when we (de)serialize from json.